docs-src/concepts/what-is-an-update.md

```diff
@@ -7,6 +7,7 @@ ssdi:
   - Released in [Temporal Server version 1.21](https://github.com/temporalio/temporal/releases/tag/v1.21.0)
   - Available in the [Go SDK](https://pkg.go.dev/go.temporal.io/sdk@v1.23.1/client#Client.UpdateWorkflowWithOptions) since [v1.23.0](https://github.com/temporalio/sdk-go/releases/tag/v1.23.0)
   - Available in the [Java SDK](https://www.javadoc.io/doc/io.temporal/temporal-sdk/latest/io/temporal/client/WorkflowStub.html#startUpdate(io.temporal.client.UpdateOptions,java.lang.Object...)) since [v1.20.0](https://github.com/temporalio/sdk-java/releases/tag/v1.20.0)
+  - Availabe in the [Python SDK](https://python.temporal.io/temporalio.workflow.html#update) since [v.1.4.0](https://github.com/temporalio/sdk-python/releases/tag/1.4.0)
```

Comment:
```suggestion
  - Available in the [Python SDK](https://python.temporal.io/temporalio.workflow.html#update) since [v.1.4.0](https://github.com/temporalio/sdk-python/releases/tag/1.4.0)
```